fronts-client/src/components/FrontsEdit/Collection.tsx

```diff
@@ -258,7 +264,10 @@ const createMapStateToProps = () => {
 			collectionSet: props.browsingStage,
 		});
 
+		const groupIds = state.collections.data[props.id]?.draft || [];
```

Comment:
I think this can be refactored such that we pass the group ids though in the collection component, rather than needing to add it to the store here. 

we have access to groups here https://github.com/guardian/facia-tool/blob/main/fronts-client/src/components/FrontsEdit/CollectionComponents/Collection.tsx#L207

My thought is we can map over this to retrieve a list of arrays, and then drill them through to the group here https://github.com/guardian/facia-tool/blob/5d6c14b39527982aac3147ab5b978124023cc4ea/fronts-client/src/components/FrontsEdit/CollectionComponents/Collection.tsx#L341

Reply:
Thanks, done - looks like it still works 🎉